cts/cts-cli.in

```diff
@@ -2230,8 +2230,11 @@ class CrmShadowRegressionTest(RegressionTest):
         ]
 
         reset_1_tests = [
+            Test("Resetting active shadow instance to active CIB requires force",
```

Comment:
requires force whether shadow instance is active or inactive, but this is probably sufficient for testing that